tests/test_jams.py

```diff
@@ -1117,3 +1117,9 @@ def _foo():
 
         # And that it says the right thing (roughly)
         assert 'deprecated' in str(out[0].message).lower()
+
+
+def test_numpy_serialize():
```

Comment:
Exactly.  I just pushed up a second test for Annotation, working on observation fields.  I think that should cover it for now.